rules-reviewed/eap7/websphere/websphere-xml.windup.xml

```diff
@@ -179,8 +179,14 @@
                 </xmlfile>
             </when>
             <perform>
-                <classification title="WebSphere deployment descriptor" effort="0">
-                    <description>This file is an auto-generated WebSphere proprietary deployment descriptor.</description>
+                <classification title="WebSphere deployment descriptor" category-id="mandatory" effort="1">
+                    <description>
+                        This file is an IBM WebSphere proprietary deployment descriptor that is typically generated when development of the EAR was done using IBM products.
+                        The XML schema documentation can be found within the WebSphere Application Server 7.0 directory under <code>/web/configDocs/packages/appdeployment/Deployment.html</code>.
+                        It should be safe to delete this file, unless it was created manually.
```

Comment:
@OndraZizka this is not valid description content. If you need to use markdown as code inline use apostroph -> \` instead of `<code>...</code>`